### PR TITLE
Describing new parameter for the tracking API and new method for the …

### PR DESCRIPTION
**Pull request by wronan**
…JS tracker

### CODE REVIEW

File: analytics/js_tracking_api.rst [hunk 1]
```diff
@@ -839,3 +839,19 @@ Example of usage::
...
+
+    _paq.push(["setUserIsAnonymous", isAnonymous]);
+
+.. describe:: isAnonymous
+
+    **Required** ``integer/boolean`` Flag that sets anonymous tracking on and off (true/false or 1/0)
+
+Example of usage::
+
+    _paq.push(["trackSiteSearch", true]);
```
Comment (azarecki) [inline analytics/js_tracking_api.rst]: Please change trackSiteSearch to setUserIsAnonymous

File: analytics/index.yaml [hunk 2]
```diff
@@ -652,6 +652,18 @@ paths:
...
           maximum: 180
           example: -74.0023705
+        - name: uia
+          in: query
+          description: |
+            Allows to track user anonymously.
+
+            When set to 1 all IP bytes will be masked (0.0.0.0), no GeoIP data will be stored
+            and visits won't be matched by the fingerprinting process. When set back to 0 (or ommited)
+            all available visitor data will be enriched.
```
Comment (RomanSek) [inline analytics/index.yaml]: When values were enumerated I was trying to describe them in a list. Check how `send_image` param is described in this file for example.

OpenAPI has also `default` attribute for params.

File: analytics/js_tracking_api.rst [hunk 3]
```diff
@@ -839,3 +839,19 @@ Example of usage::
...
+
+Tracking user anonymously
+``````````````````````````
+To track visitor anonymously (without consent) use the ``setUserIsAnonymous`` function::
+
+    _paq.push(["setUserIsAnonymous", isAnonymous]);
+
+.. describe:: isAnonymous
+
+    **Required** ``integer/boolean`` Flag that sets anonymous tracking on and off (true/false or 1/0)
```
Comment (RomanSek) [inline analytics/js_tracking_api.rst]: I think that alternative types are usually separated by `|` character (e.g. in [JSDocs](http://usejsdoc.org/tags-type.html)), but I see that this character is also used in our docs.

Try to format JS values in descriptions. Check how it's done in `checkOnScroll` description in this file.

File: analytics/js_tracking_api.rst [hunk 4]
```diff
@@ -839,3 +839,19 @@ Example of usage::
...
+
+Example of usage::
+
+    _paq.push(["trackSiteSearch", true]);
+
```
Comment (RomanSek) [inline analytics/js_tracking_api.rst]: I see that RST format has a [versionadded](http://www.sphinx-doc.org/en/stable/markup/para.html#directive-versionadded) directive to notify user when element was introduced. Maybe we should use that at the end of description? We didn't use it yet, but you can find `deprecated` directive in this file that conveys similar information.

File: analytics/js_tracking_api.rst (hunk 1 above)
Comment (wronan) [reply analytics/js_tracking_api.rst]: F***...

File: analytics/js_tracking_api.rst (hunk 3 above)
Comment (wronan) [reply analytics/js_tracking_api.rst]: Hmm... setEcommerceView parameters description mislead me.

Comment (wronan) [reply analytics/js_tracking_api.rst]: I'll keep it simple and stick with boolean.

**Pull request merged**